tectonic/synctex.c

```diff
@@ -693,7 +605,7 @@ void synctex_horizontal_rule_or_glue(int32_t p, int32_t this_box __attribute__ (
             }
             break;
         default:
-            printf("\nSynchronize ERROR: unknown node type %i\n", SYNCTEX_TYPE(p));
+            ttstub_issue_warning("\nSynchronize ERROR: unknown node type %i\n", SYNCTEX_TYPE(p));
```

Comment:
This should be `ttstub_issue_error` with message "unknown node %d in SyncTeX". Leading and trailing newlines are not needed with this function.